src/styles/globalStyles.tsx

```
@@ -0,0 +1,106 @@
+`;
+
+const pulseGentle = keyframes`
+  0%, 100% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.5;
+  }
+`;
+
+export const globalStyles = css`
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)